A Semi-non-linear Bastion Factory

ribagi opened the pull request:
New items from a few unused factories. Walls from Fancy Ore semlter, Base from Smelter, Gearbox from Iron Forge, Object d'art from Stain Glass, Flooring from Stone Brick, Rations from Bakery, Smaragdus from Iron Cauldron, Silicon Tetranitratobihydrotrioxycarbon from Explosive, and Pure Ice from Crystallisation Factory

Commit:
A Semi-non-linear Bastion Factory 

New items from a few unused factories. Walls from Fancy Ore semlter, Base from Smelter, Gearbox from Iron Forge, Object d'art from Stain Glass, Flooring from Stone Brick, Rations from Bakery, Smaragdus from Iron Cauldron, Silicon Tetranitratobihydrotrioxycarbon from Explosive, and Pure Ice from Crystallisation Factory

Commit:
Update config.yml

Commit:
Update config.yml

Commit:
Update config.yml

Commit:
Update config.yml

ttk2 commented:
and you put all of those together for the bastion factory?

ribagi commented:
Yea

pruby commented:
I really like the direction here - collecting inputs from the other factories. It provides the difficulty beyond cost that my recipe lacks. However, it seems way too cheap if you already have the other factories, which a lot of towns do. Total cost 48d in diamonds, 4d of exp, then a few other bits and pieces which at an extreme stretch could bring it up to 64d, then 1d per bastion.  In my opinion it needs to be much higher than that, by about 8-10 times.

Also need to touch up indentation in a couple of places. YAML is indentation sensitive - keys at the same level _must_ have the same indentation. In particular "outputs" needs to be on the same indentation as "inputs", and line 6835 looks indented too far (next line should be indented, is on same level).

ribagi commented:
The costs I pulled out of the air. Even though when I did put them out, I did feel that they were on the low side. It is also on a side note that the cheaper items are only there to server as a check if you have a factory or not. The methodology on when that happens is quite random.

On the formatting thing, one thing I hate about notepad --> github is that while it looks perfect in notepad, when you send a pull request everything goes to shit.

Commit:
Formatting stuff

Commit:
Update config.yml

Commit:
Update config.yml

ttk2 commented:
so up the cost, get it formatted and then merge this one it is?

On Wed, Mar 26, 2014 at 4:43 PM, ribagi notifications@github.com wrote:

> The costs I pulled out of the air. Even though when I did put them out, I
> did feel that they were on the low side.
> 
> On the formatting thing, one thing I hate about notepad --> github is that
> while it looks perfect in notepad, when you send a pull request everything
> goes to shit.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/ttk2/FactoryMod/pull/19#issuecomment-38743989
> .

pruby commented:
Still tabs mixed through this - will patch and add extra costs.

Commit:
Update config.yml

x8 item 'Base' cost, x4 item 'Walls' cost, and x2 item 'Smaragdus' cost to the Final Bastion Factory.

The Emerald Block cost to Smaragdus increased by x3.

Commit:
Update config.yml

ribagi commented:
x8 item 'Base' cost, x4 item 'Walls' cost, and x2 item 'Smaragdus' cost to the Final Bastion Factory.

The Emerald Block cost to Smaragdus increased by x3.

Some minor fixes too.

pruby commented:
Looks like we were amending at the same time - added a version on https://github.com/ttk2/FactoryMod/pull/21 with docs & cost breakdown. I've increased the costs _way_ further than that.

pruby commented on config.yml:
```diff
@@ -2288,6 +2297,68 @@ production_factories:
+        material: COAL
+        durability: 1
+    inputs:
+	  Silicon Tetranitratobihydrotrioxycarbon:
```
These lines are still tab-indented.

pruby commented on config.yml:
```diff
@@ -6542,4 +6613,277 @@ production_recipes:
+      Sponge:
+        material: SPONGE
+        amount: 64     
+	   
```
And a final tab here. They're dangerous in YAML because you can't see the difference, but the parser does.
